pp_test.go

```diff
@@ -35,3 +35,15 @@ func TestColorScheme(t *testing.T) {
 		t.FailNow()
 	}
 }
+
+func TestWithLineInfo(t *testing.T) {
+	WithLineInfo(true)
+	if withLineInfo != true {
```

Comment:
I think this test won't save us from bugs. The function is very simple and it will be just Go compiler's test. It makes test code long and increase maintenance cost. Ideally we should write test of just output. WDYT?